osu.Game/Screens/Menu/Button.cs

```diff
@@ -259,7 +259,7 @@ protected override bool OnKeyDown(InputState state, KeyDownEventArgs args)
         {
             if (args.Repeat) return false;

```

Comment:
probably cleaner to add a new `if (state.Keyboard.ControlPressed || state.Keyboard.ShiftPressed || state.Keyboard.AltPressed) return false` here rather than add on the end of the affirmative conditional